terraform/outputs.tf

```diff
@@ -0,0 +1,59 @@
+output "aws_region" {
+  value = "${var.aws_region}"
+}
+
+output "aws_az1" {
+  value = "${var.aws_az1}"
+}
+
+output "aws_az2" {
+  value = "${var.aws_az2}"
+}
+
+output "mgmt_vpc_name" {
+  value = "${var.mgmt_vpc_name}"
+}
+
+output "mgmt_vpc_id" {
+  value = "${module.mgmt-vpc.vpc_id}"
+}
```

Comment:
Do dashes work in Terraform references? Seems to break the syntax highlighter, at least. I'd suggest using underscores, for consistency.